<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilmMarkNInfoWindClass</class>
 <widget class="QDialog" name="FilmMarkNInfoWindClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>189</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FilmMarkNInfoWind</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="pg_addMark">
    <widget class="QLabel" name="lb_mark">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Film Mark:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnEd_addMark">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>0</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pg_addMovieWatchInfo">
    <widget class="QLineEdit" name="lnEd_hour">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lb_hour">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Hour:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lnEd_min">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lb_min">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Minute:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pg_SerialWatchInfo">
    <widget class="QLineEdit" name="lnEd_episode">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lb_episode">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Episode:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pB_ok">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_cancel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_action">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ink Free</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Action</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pB_cancel</sender>
   <signal>clicked()</signal>
   <receiver>FilmMarkNInfoWindClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
